getting-started/php/README.MD

```diff
@@ -1 +1 @@
-[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://che.openshift.io/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/getting-started/php/devfile.yaml)
+[![Contribute](https://www.eclipse.org/che/contribute.svg)](https://workspaces.openshift.com/f?url=https://raw.githubusercontent.com/redhat-developer/devfile/master/getting-started/php/devfile.yaml)
```

Comment:
verified workspace startup and LS basic functionality - OK
Command - startup works correctly, but there are failures with stop and restart.
![Screenshot from 2021-02-22 13-22-57](https://user-images.githubusercontent.com/29278800/108708036-30e2dd00-7511-11eb-8990-3bea9f32a547.png)